config/services.yaml

```diff
@@ -4,7 +4,10 @@
 # Put parameters here that don't need to change on each machine where the app is deployed
 # https://symfony.com/doc/current/best_practices.html#use-parameters-for-application-configuration
 parameters:
-
+    baseUri: '%env(BASE_URI)%'
```

Comment:
i think baseUrl can be just a constant since this value will not change for others or for production